netty-server/docker-compose.yml

```diff
@@ -19,24 +19,30 @@ services:
     command: '--data-dir /data --api-key=xyz --enable-cors'
     
   formkiq:
-    image: formkiq/api-server:1.13.0-SNAPSHOT-20231125
+    image: docker.io/formkiq/api-server:latest
```

Comment:
Was not sure what to put here, since Docker image with this code is not yet available.